### DISCUSSION TITLE
VSCode extension : i18n json editor

### DISCUSSION THREAD

**Discussion by thibault-vanderseypen** [ocombe/ng2-translate#1360] [Show and tell]
Hello fellows, 

I developed an extension to edit the i18n json files directly into VSCode. 
If you use this IDE for your projects don't hesitate to have a try : https://marketplace.visualstudio.com/items?itemName=thibault-vanderseypen.i18n-json-editor. You just have to make a right click on your i18n folder and select "**i18n JSON Editor**" to open an editor. 

Have a good day,

Thibault

**Comment by rbalet** [comment]
The following vs-code extension seems to be better supported and also handle missing keys https://marketplace.visualstudio.com/items?itemName=Lokalise.i18n-ally .

Closing this topic
